src/foraging_gui/Foraging.py

```diff
@@ -4120,6 +4131,66 @@ def _open_mouse_on_streamlit(self):
                          '&session_plot_mode=all+sessions+filtered+from+sidebar'
                          '&session_plot_selected_draw_types=1.+Choice+history'
         )
+    
+    def _generate_upload_manifest(self,FIP=False):
+        '''
+            Generates a manifest.yml file for triggering data copy to VAST and upload to aws
+        '''
+        try: 
+            if not hasattr(self, 'project_name'):
+                self.project_name = 'Behavior Platform'
+            
+            if FIP: ## DEBUG, need to figure out how to set this flag.  
+                schedule = self.acquisition_datetime.split(' ')[0]+' 23:59:00'
+                capsule_id = 'c089614a-347e-4696-b17e-86980bb782c' 
+                mount = self.session_name 
+            else:
+                schedule = self.acquisition_datetime.split(' ')[0]+' 23:59:00'
+                capsule_id = 'c089614a-347e-4696-b17e-86980bb782c' 
+                mount = self.session_name
+ 
+            # Define contents of manifest file
+            contents = {
+                'acquisition_datetime': self.acquisition_datetime,
+                'name': self.session_name,
+                'platform': 'behavior',
+                'subject_id': int(self.ID.text()),
+                'capsule_id': capsule_id,
+                'mount':mount,
```

Comment:
Let's set `mount == "FIP"` for now.

Reply:
Yup, with Ahad, we confirmed `FIP` works.

Reply:
Should we use `FIP` for behavior only sessions as well?

Reply:
I set both to 'FIP' for now. but that seems odd

Reply:
for now Ahad orgnized the code so that pure behavior sessions (recognized by session.json modality) will be processed by the capsule too and be nwb-packaged too. 

For that mount has to be `FIP` for now.

Reply:
![Screenshot 2024-07-01 at 16 09 03](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/assets/54397345/9b7311e1-c7ff-434e-a691-52240368dcc1)

Earlier communication with David on this